script.js

```diff
@@ -1,5 +1,7 @@
 $(document).ready(function () {
     randomDrink();
+    // on click event for classic drink btns
+    $(document).on("click", ".drink", classicDrinkButtons);
```

Comment:
I like putting the stuff actual stuff that's happening near each other rather than doing `randomDrink()` and then the document stuff after all the functions are declared.